platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/PlatformDataTests.java

```diff
@@ -1,150 +0,0 @@
-        ConstructableRegistry.getInstance().registerConstructables("com.swirlds");
-    }
-
-    private static PlatformData generateRandomPlatformData(final Random random) {
-        final int randomBound = 10_000;
-
-        final List<MinimumJudgeInfo> minimumJudgeInfo = new LinkedList<>();
-        final int minimumJudgeInfoSize = random.nextInt(1, MinimumJudgeInfo.MAX_MINIMUM_JUDGE_INFO_SIZE);
-        for (int i = 0; i < minimumJudgeInfoSize; i++) {
-            minimumJudgeInfo.add(new MinimumJudgeInfo(random.nextLong(randomBound), random.nextLong(randomBound)));
-        }
-
-        return new PlatformData()
-                .setRound(random.nextLong(randomBound))
-                .setHashEventsCons(randomHash(random))
-                .setConsensusTimestamp(Instant.ofEpochSecond(random.nextInt(randomBound)))
-                .setCreationSoftwareVersion(new BasicSoftwareVersion(random.nextInt(randomBound)))
-                .setEpochHash(randomHash(random))
-                .setSnapshot(new ConsensusSnapshot(
-                        random.nextLong(),
-                        List.of(randomHash(random), randomHash(random), randomHash(random)),
-                        minimumJudgeInfo,
-                        random.nextLong(),
-                        randomInstant(random)));
-    }
-
-    @Test
-    @DisplayName("Serialization Test")
-    void serializationTest() throws IOException {
-        final Random random = getRandomPrintSeed();
-        final PlatformData platformData = generateRandomPlatformData(random);
-
-        final ByteArrayOutputStream byteOut = new ByteArrayOutputStream();
-        final SerializableDataOutputStream out = new SerializableDataOutputStream(byteOut);
-
-        out.writeSerializable(platformData, true);
-
-        final SerializableDataInputStream in =
-                new SerializableDataInputStream(new ByteArrayInputStream(byteOut.toByteArray()));
-
-        final PlatformData deserialized = in.readSerializable();
-
-        CryptographyHolder.get().digestSync(platformData);
-        CryptographyHolder.get().digestSync(deserialized);
-
-        assertEquals(platformData.getHash(), deserialized.getHash(), "hash should match");
-    }
-
-    @Test
-    @DisplayName("Serialization Test")
-    void equalityTest() {
-        final Random random = getRandomPrintSeed();
-
-        final long seed1 = random.nextLong();
-        final long seed2 = random.nextLong();
-
-        final PlatformData platformData1 = generateRandomPlatformData(new Random(seed1));
-        final PlatformData platformData1Duplicate = generateRandomPlatformData(new Random(seed1));
-        final PlatformData platformData2 = generateRandomPlatformData(new Random(seed2));
-
-        assertEquals(platformData1, platformData1Duplicate, "should be equal");
-        assertNotEquals(platformData1, platformData2, "should not be equal");
-    }
-
-    @Test
-    @DisplayName("Update Epoch Hash Test")
-    void updateEpochHashTest() {
```

Comment:
There are a few tests in this class that did not get implemented in the `PlatformStateTests` class when the functionality was moved over. Could you please move these tests over to the other class?

Reply:
Sure, will do.

Reply:
Done